### DISCUSSION TITLE
Visibility of Notes field?

### DISCUSSION THREAD

**Discussion by plk** [photoprism/photoprism#3933] [General]
I can see the Notes field when I edit a picture but is there a way to see this on one of the main folder views? Or perhaps a way of viewing all tags for an image without editing them?

**Comment by lastzero** [comment]
Use the description field for this 👍

**Reply by plk** [reply]
I would but I need line breaks and in the main view, it won't show line breaks ... tried HTML, `\n` etc. no luck.

**Comment by plk** [comment]
I just realised that if you click on the description, you get all the tags which is good, thanks.
